Orange/widgets/data/owselectrows.py

```diff
@@ -231,7 +248,9 @@ def set_new_values(self, oper_combo, adding_all, selected_values=None):
         #         child.setParent(None)
 
         def add_textual(contents):
-            le = gui.lineEdit(box, self, None)
+            le = gui.lineEdit(box, self, None,
+                              sizePolicy=QSizePolicy(QSizePolicy.Expanding,
++                                                     QSizePolicy.Expanding))
```

Comment:
You missed a plus here. The widget still works because +0x7 is still 0x7, but it's better without it. Note, you can more easily apply diff patches with `git-apply`.